File: dev/requirements/screen/cases/venue-line.2.3.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 2.3 — the room's name across the top, upper-cased and letter-spaced.
let venueLine_2_3 = RequirementCase(id: "2.3") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "venue-line", id: "2.3", region: .venue)
    }
}
#endif

File: dev/requirements/screen/cases/venue-line.2.3.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 2.3 — The header names the room, upper-cased and letter-spaced.
final venueLine_2_3 = RequirementCase('2.3', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.venue,
    slug: 'venue-line',
    id: '2.3',
  );
});
